README.md

```diff
@@ -66,6 +66,17 @@ Getting Mosh
   sure that a running process's environment variables are not readable by other
   users.  We have confirmed that this is the case on GNU/Linux, OS X, and
   FreeBSD.
+  
+Building Mosh From Source
+------------
+  On most operating systems, building is simple. Simply run the following commands as root:
```

Comment:
It not recommended to build software as root. Only the `make install` step needs root access (and then only if you are installing to a root-owned directory such as the default `/usr/local`).